src/extension.ts

```diff
@@ -148,6 +150,23 @@ export async function activate(context: vscode.ExtensionContext) {
             () => { vscode.env.openExternal(vscode.Uri.parse(githubUrl)) }
         )
 
+        vscode.commands.registerCommand(
+            'aws.welcome',
+            async () => {
+                const view = vscode.window.createWebviewPanel(
```

Comment:
Prefer to make this a standalone function, then provide the function to the registerCommand call.
The `activate` method needs cleanup, but we can manage the growth in line count.

Reply:
✔️